app/models/katello/host/info_provider.rb

```diff
@@ -28,7 +28,7 @@ def content_view_info
           'label' => host.content_view.try(:label),
           'latest-version' => host.content_view.try(:latest_version),
           'version' => content_version.try(:version),
-          'published' => content_version.try(:created_at).try(:time),
+          'published' => content_version.try(:created_at).try(:time).to_s,
```

Comment:
should this be .try(:to_s)?  otherwise you'd end up with 'nil'  right?

Reply:
```
[1] pry(main)> nil.to_s
=> ""